src/test/java/com/fivetran/agent/mysql/BatchUpdaterTest.java

```diff
@@ -188,12 +192,10 @@ public void binlogSync_updateTableDefinition() throws Exception {
         TableState tableState = new TableState();
 
         tableState.finishedImport = true;
-        state.tables.put(tableRef, tableState);
+        state.tableStates.put(tableRef, tableState);
+        state.tableDefinitions.put(tableRef, initialTableDef);
```

Comment:
in `updateStateTableInfo()`we can always expect the number of TableStates to equal the number of TableDefinitions